tests/storage-metadata

```diff
@@ -229,6 +229,17 @@ EOF
         lxc move c1:i1 i1
         [ "$(lxc query "/1.0/instances/i1?recursion=1" | jq '.snapshots | length')" = "1" ]
 
+        echo "==> Check the migrated ${type} can be started"
+        lxc start i1
+        waitInstanceBooted i1
```

Comment:
Is that really needed? If you really require it to be somewhat functional there is a slightly lighter version: `waitInstanceReady`.